### PR TITLE
fix configuration file links

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/camunda-docs/issues/3744.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Review by akeller — APPROVED** [review]
Thanks for fixing this per version! Can you add a reminder to update the link when we cut the docs release for 8.6? 

I thought there was something in the backlog for exploring variables for this, but I'm not seeing anything.

File: versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md [hunk 1]
```diff
@@ -31,10 +31,10 @@ The default configuration is not suitable for a standalone gateway node. To run
...
-- [`config/broker.standalone.yaml.template` Standalone Broker (with embedded gateway)](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/broker.standalone.yaml.template) - Complete configuration template for a standalone broker with embedded gateway. Use this as the basis for a single broker deployment for test or development.
-- [`config/broker.yaml.template` Broker Node (without embedded gateway)](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/broker.yaml.template) - Complete configuration template for a broker node without embedded gateway. Use this as the basis for deploying multiple broker nodes as part of a cluster.
-- [`config/gateway.yaml.template`](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/gateway.yaml.template) - Complete configuration template for a standalone gateway.
+- [`config/application.yaml` Standalone Broker (with embedded gateway)](https://github.com/camunda/zeebe/blob/8.2.0/dist/src/main/config/application.yaml) - Default configuration containing only the most common configuration settings. Use this as the basis for a single broker deployment for test or development.
```
Comment (falko) [inline versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md:34]: Is there no stable 8.2.x branch anymore that would always point to the latest patch for that minor version?

**Review by falko — CHANGES_REQUESTED** [review]
Micro releases should usually not add features to the configuration files. However, patches for bugs could sometimes be necessary. So, I'd feel more comfortable with linking to the version branch.

File: versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md (hunk 1 above)
Comment (falko) [reply versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md:34]: If I understand it correctly, the stable branches may could in worst case contain an unreleased backported bugfix but they would be an easy target to link to, which doesn't need changing with every patch release, e.g. https://github.com/camunda/zeebe/blob/stable/8.2/dist/src/main/config/application.yaml

Comment (christinaausley) [reply versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md:34]: For consistency with https://github.com/camunda/camunda-docs/pull/3743/files, we'll keep with a layout like `https://github.com/camunda/zeebe/tree/8.2.0/exporters/elasticsearch-exporter/src/main/resources`, with the exception of the `next` version, which will be updated for each release 👍 

Will add this set of links to my notes for release updates.

**Comment by christinaausley** [comment]
Will resolve merge conflicts in https://github.com/camunda/camunda-docs/pull/3829 👍

**Pull request merged**